packages/terra-site/src/examples/table/examples/MultiSelect.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import Table from 'terra-table';
+
+const table = () => (
+  <div>
+    <h2>Examples</h2>
+    <h3>Multiple Selectable Rows</h3>
+    <Table isStriped={false}>
+      <Table.Header>
+        <Table.HeaderCell content={'Name'} key={'NAME'} minWidth={'small'} />
+        <Table.HeaderCell content={'Address'} key={'ADDRESS'} minWidth={'medium'} />
+        <Table.HeaderCell content={'Phone Number'} key={'PHONE_NUMBER'} minWidth={'large'} />
+      </Table.Header>
+      <Table.MultiSelectableRows>
+        <Table.Row key={'PERSON_0'}>
+          <Table.Cell content="John Smith" key="NAME" />
+          <Table.Cell content="123 Adams Drive" key="ADDRESS" />
+          <Table.Cell content="111-222-3333" key="PHONE_NUMBER" />
+        </Table.Row>
+        <Table.Row key={'PERSON_1'}>
+          <Table.Cell content="Jane Smith" key="NAME" />
+          <Table.Cell content="321 Drive Street" key="ADDRESS" />
+          <Table.Cell content="111-222-3333" key="PHONE_NUMBER" />
+        </Table.Row>
+        <Table.Row key={'PERSON_2'}>
+          <Table.Cell content="Dave Smith" key="NAME" />
+          <Table.Cell content="213 Raymond Road" key="ADDRESS" />
+          <Table.Cell content="111-222-3333" key="PHONE_NUMBER" />
+        </Table.Row>
+      </Table.MultiSelectableRows>
+    </Table>
+    <h3>Multiple Selectable Rows (Max Selectable: 2)</h3>
+    <Table isStriped={false}>
+      <Table.Header>
+        <Table.HeaderCell content={'Name'} key={'NAME'} minWidth={'small'} />
+        <Table.HeaderCell content={'Address'} key={'ADDRESS'} minWidth={'medium'} />
+        <Table.HeaderCell content={'Phone Number'} key={'PHONE_NUMBER'} minWidth={'large'} />
+      </Table.Header>
+      <Table.MultiSelectableRows maxSelectionCount={2}>
+        <Table.Row key={'PERSON_0'}>
+          <Table.Cell content="John Smith" key="NAME" />
+          <Table.Cell content="123 Adams Drive" key="ADDRESS" />
+          <Table.Cell content="111-222-3333" key="PHONE_NUMBER" />
+        </Table.Row>
+        <Table.Row key={'PERSON_1'}>
+          <Table.Cell content="Jane Smith" key="NAME" />
+          <Table.Cell content="321 Drive Street" key="ADDRESS" />
+          <Table.Cell content="111-222-3333" key="PHONE_NUMBER" />
+        </Table.Row>
+        <Table.Row key={'PERSON_2'}>
+          <Table.Cell content="Dave Smith" key="NAME" />
+          <Table.Cell content="213 Raymond Road" key="ADDRESS" />
+          <Table.Cell content="111-222-3333" key="PHONE_NUMBER" />
+        </Table.Row>
+      </Table.MultiSelectableRows>
+    </Table>
+    <h3>Multiple Selectable Rows (First Row Initially Selected)</h3>
```

Comment:
We probably don't need this site example. The Table site page shows initially shows a pre-selected table row which provides an example of this behavior. If you felt it was still important to show within mulit-select rows, maybe update an one of the other examples?

Reply:
I just added it to the default MultiSelectableRows example. The other examples showed that behavior before so seemed like it would work here =)